src/qt/overviewpage.cpp

```diff
@@ -283,7 +283,7 @@ void OverviewPage::on_btnWebsite_pressed()
 
 void OverviewPage::on_btnBX_pressed()
 {
-    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/"));
+    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));
```

Comment:
I think the slash in front of the pound should be removed:

-> `/block#pk_campaign`